Modules/Registration/RegistrationMethodsv4/include/itkImageRegistrationMethodv4.h

```diff
@@ -532,7 +532,7 @@ class ITK_TEMPLATE_EXPORT ImageRegistrationMethodv4 : public ProcessObject
   SetMetricSamplePoints();
 
   SizeValueType m_CurrentLevel;
```

Comment:
Wouldn't it make sense to move all of [these ivar initializations](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3.0/Modules/Registration/RegistrationMethodsv4/include/itkImageRegistrationMethodv4.hxx#L57-L63) here?

Reply:
https://github.com/InsightSoftwareConsortium/ITK/pull/3845#discussion_r1060878875 Wops you're right. Thanks for the heads-up @dzenanz ! Amended the commit to initialize it to 0 in 54367db.

> Wouldn't it make sense to move all of [these ivar initializations](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3.0/Modules/Registration/RegistrationMethodsv4/include/itkImageRegistrationMethodv4.hxx#L57-L63) here?

That is a style change. I used the initialization list to fix the bug because the initialization trend that is being pushed in the code base is that one. IMHO best would be to propose the style change you propose in a separate PR. I could do a separate commit within this PR, but I prefer the bug to be fixed and merged as soon as possible.

I also realize that the implementation file contains method documentation blocks. Following the previous comment, best would be to do a separate PR to address that.

Anybody is welcome to contribute with them.